runtime/queries/c/highlights.scm

```diff
@@ -73,20 +87,23 @@
 (call_expression
   function: (field_expression
     field: (field_identifier) @function))
+(function_declarator) @function
 (function_declarator
-  declarator: (identifier) @function)
+  parameters: (parameter_list) @variable.parameter)
 (preproc_function_def
   name: (identifier) @function.special)
 
+(compound_statement) @variable
+(init_declarator) @variable
```

Comment:
These two are covered by the deleted

```scm
(identifier) @variable
```

stanza below. That should be restored and these two lines should be removed. With these, entire function bodies and initialization statements inherit variable theming (to see the effect, set "variable" to have `modifiers = [ "italics" ]` in your theme).

Reply:
You can move the `; inherits: c` to the bottom of `runtime/queries/cpp/highlights.scm` so that those stanzas have lower precedence than any stanzas in the cpp highligts

Reply:
Doing that ☝️ should also fix this case:

```cpp
this->b<C, D>();
//    ^ @variable.other.member but should be @function
```

Reply:
Ah I didn't realise that was possible, that's much simpler. Done.